app/index.html

```diff
@@ -15,6 +15,13 @@
       rel="stylesheet"
       href="{{rootURL}}assets/website-my.css"
     />
+    <link
```

Comment:
We have stylesheet for cropper too , nice!!!

Reply:
Yes, its good, made it task easy